tools/lsp/build.rs

```diff
@@ -2,5 +2,13 @@
 // SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-Royalty-free-1.1 OR LicenseRef-Slint-commercial
 
 fn main() {
+    // Make the compiler handle ComponentContainer:
+    // TODO: Remove if ComponentContainer anc component-factory no longer need this!
+    std::env::set_var("SLINT_ENABLE_EXPERIMENTAL_FEATURES", "1");
+    slint_build::compile_with_config(
+        "ui/main.slint",
+        slint_build::CompilerConfiguration::new().with_style("fluent-light".into()),
```

Comment:
I don't think we should force the style.  IMHO it should use the native style, as we want the preview UI to look native.

Reply:
We should be a model and use the native styling here

Reply:
At the moment we have no dark style for slintpad. And I think it doesn't look good display a dark variant of the preview ui inside the light only slintpad. That was the reason. And the long time goal should be to display it in the native style.

Reply:
True, we can set the SLINT_STYLE env variable when compiling for slintpad.
But this preview is also used for the native vscode extension.  And there we want to use the native style.

Reply:
Make sense to me.